ui/src/components/VoxPrompts.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html} from 'lit-element';
+
+import promptApi from '../utils/PromptApi';
+
+export class VoxPrompts extends LitElement {
+    static get properties() {
+        return {
+            prompt: {type: String},
+        };
+    }
+    constructor() {
+        super();
+        this.getNewPrompt();
+    }
+
+    getNewPrompt() {
+        promptApi
+            .getNewPrompt()
+            .then((nextPrompt) => (this.prompt = nextPrompt));
+    }
+
+    handleSkip(e) {
+        this.getNewPrompt();
```

Comment:
I think `this.getNewPrompt` fits better in the `firstUpdated` LitElement lifecycle event. Usually constructors in these components don't do anything except set initial values.

Reply:
thanks lol I was looking at the docs and wasn't really sure if firstUpdated really meant that it'd call it on first update